server/controllers/doppler.controller.js

```diff
@@ -48,6 +49,22 @@ class DopplerController {
         await redis.quitAsync();
       }
     }
+    
+    async uninstallShop({ dopplerData: { isSuperUser }, body: { shop } }, response) {

```

Comment:
From where do we get the store's name

Reply:
I understand that the shop property comes in the body request
_{ dopplerData: { isSuperUser }, body: { shop } }_ is the parameter and represent de request

Reply:
Paso a castellano, nosotros tenemos que enviar ese request con un dato dentro del body del request. Ese dato que es? La URL de la tienda de shopify? un nombre que viene de otro request?

Reply:
Es el dominio de la tienda.